extensions/websockets-next/pom.xml

```diff
@@ -12,6 +12,8 @@
 
     <artifactId>quarkus-websockets-next-parent</artifactId>
     <name>Quarkus - WebSockets Next</name>
+    <description>This extension provides a modern declarative API to define WebSocket server and client endpoints.</description>
```

Comment:
I think we usually try to start with a verb and we'd better avoid `This extension provides...`.

Reply:
Adjusted and squashed